### ISSUE TITLE
Problems Starting Up

### ISSUE THREAD

**Issue by CaptianBoo** [TheLimePixel/Modern-Skyblock-3#822]
**Describe the bug**

**To Reproduce**
1. Start Up

**Crash Reports**
It said crash but did not supply a report

**Pack Information**
 - Pack version: 3.5.0
 - Pack Mode: Didn't even start
 - SP or MP: ?
 - Added Mods: None

**Screenshots**
(https://i.postimg.cc/cCcM14h7/Untitled.png)

**Additional context**
Dell Laptop; 5000 Series

**Comment by FruityPewps** [comment]
Probably not enough RAM. How much are you allocating?

**Comment by CaptianBoo** [comment]
Sorry this was a late response, I got your response late for me and I woke up at 8-something, and since I get a bit iffy about things that I don't use on my laptop I had to re-download. 

In any-case it was set to system settings.

**Comment by FruityPewps** [comment]
So are you good now?

**Comment by CaptianBoo** [comment]
Somewhat, I bumped it up to 6 exactly as it stayed in my recommended and it would stuttering once in a while, right now I have it at 6,040 and it seems better

**Comment by FruityPewps** [comment]
Yeah try using some good garbage collecting arguments if you want. Can google that. This pack sits more comfortable around 8. Personally I run 12.

**Comment by CaptianBoo** [comment]
A crash happened when I bumped it to 8 
[https://gist.github.com/CaptianBoo/ae3d4ba9f295658a6c4c63200147cebd](Crash)

**Comment by FruityPewps** [comment]
Link is not working.
